PSReadLine/Render.cs

```diff
@@ -799,6 +799,40 @@ private int ConvertLineAndColumnToOffset(Point point)
             return (point.Y == y) ? offset : -1;
         }
 
+        /// <summary>
+        /// Returns the line number under the cursor.
+        /// </summary>
+        /// <returns></returns>
```

Comment:
Remove empty doc comment.

Reply:
You mean the empty `returns` clause, or the whole comment ?

Reply:
Just the `returns` clause - it's just noise with actual documentation.